views/page/course/module/events/attendee.html

```diff
@@ -24,15 +25,14 @@ <h3 class="govuk-heading-m">Attendee</h3>
                                     {% if booking.status === "Requested" %}
                                         Pending
                                     {% elif booking.status === "Confirmed" %}
-                                        Booked
+                                        Confirmed
```

Comment:
Can we i18n these values? I've seen them in a few places and it will be helpful if we need to change text in future.